core/templates/dev/head/dashboard/notifications_dashboard.html

```diff
@@ -26,46 +26,53 @@ <h2 class="oppia-page-heading">Notifications</h2>
       Last updated: <[getLocaleAbbreviatedDatetimeString(jobQueuedMsec)]>
     </div>
 
-    <md-card class="oppia-page-card oppia-notifications-dashboard-card">
-      <div ng-if="recentNotifications.length === 0">
+    <div class="oppia-notifications-tray">
+      <md-card ng-if="recentNotifications.length === 0" class="oppia-page-card oppia-notifications-dashboard-card">
         <em>No recent notifications.</em>
-      </div>
+      </md-card>
+
+      <div ng-if="recentNotifications.length > 0" >
+        <md-card ng-repeat="notification in recentNotifications"
+            ng-class="{'oppia-dashboard-row-recent': lastSeenMsec && lastSeenMsec < notification.last_updated_ms && notification.author_username !== currentUsername, 'oppia-dashboard-row': !lastSeenMsec || lastSeenMsec >= notification.last_updated_ms || notification.author_username === currentUsername}"
+            ng-click="navigateToItem(notification.activity_id, notification.type)">
+            <div ng-if="notification.type === 'feedback_thread'" class="oppia-feedback-notification">
+              <div>
+                <span class="label label-primary"
+                      ng-if="lastSeenMsec && lastSeenMsec < notification.last_updated_ms && notification.author_username !== currentUsername">
+                  New
+                </span>
+              </div>
+              <div class="oppia-notification-type">
+                <span class="glyphicon glyphicon-comment"
+                      title="Feedback thread"></span>
+              </div>
+                <div>
+                  <div class="oppia-notification-author-username"><strong><a ng-click="navigateToProfile($event, notification.author_username)"><[notification.author_username]></a><span ng-if="!notification.author_username">Anonymous</span></strong> left a comment on<span class="oppia-notification-activity-title"><[notification.activity_title]></span></div>
+                </div>
+              <div class="oppia-notification-last-updated"><[getLocaleAbbreviatedDatetimeString(notification.last_updated_ms)]></div>
+              <div class="clearfix"></div>
```

Comment:
IMHO clearfix is a better way for handling floating elements, and in this particular case, this would suffice. Also, if we remove this, we need to specify the height for the notification cards for multiple screen size. They won't scale with the flow of content.

If you think we should remove it then we'll remove it and figure another way around. Please let me know the needful.